meta-flex-support/recipes-core/images/development-image.bb

```diff
@@ -5,22 +5,22 @@
 SUMMARY = "A development/debugging image that fully supports the target \
 device hardware."
 
-IMAGE_FEATURES = "${EXTRA_IMAGE_FEATURES}"
-EXTRA_IMAGE_FEATURES = "debug-tweaks codebench-debug ssh-server-openssh"
-IMAGE_FEATURES:append:feature-tracing = " tools-profile"
+IMAGE_FEATURES = " \
+    codebench-debug \
+    debug-tweaks \
+    package-management \
+    ssh-server-openssh \
+"
```

Comment:
Have you check the default value of EXTRA_IMAGE_FEATURES if it is bringing any other package? In that case, we'll have to empty that variable as well.

Reply:
I checked the environment, this is not an issue. This PR is good to merge.